datadog/getmetrics_datadog.py

```diff
@@ -215,7 +215,9 @@ def get_metric_data(metric_list, host_list, start_time, end_time, collected_data
 
     def format_data_entry(json_data_entry):
         metric_name = json_data_entry.get('metric')
+        logger.debug("json_data_entry is: " + str(json_data_entry))
```

Comment:
how much data will this end up printing to screen? Should we print every entry, every 10 entries, every N entries?

Reply:
example for json_data_entry: 

json_data_entry is: {u'end': 1556723816000, u'display_name': u'system.mem.total', u'metric': u'system.mem.total', u'interval': 1, u'start': 1556723779000, u'length': 11, u'query_index': 6, u'aggr': None, u'attributes': {}, u'expression': u'system.mem.total{js-Gazelle:N/A}', u'scope': u'js-Gazelle:N/A', u'unit': [{u'scale_factor': 1.0, u'name': u'byte', u'family': u'bytes', u'short_name': u'B', u'plural': u'bytes', u'id': 2}, None], u'pointlist': [[1556723779000.0, 16559710208.0], [1556723782000.0, None], [1556723786000.0, None], [1556723789000.0, None], [1556723794000.0, 16559710208.0], [1556723797000.0, None], [1556723801000.0, None], [1556723804000.0, None], [1556723809000.0, 16559710208.0], [1556723812000.0, None], [1556723816000.0, None]]}

I think it does not have much.